moped-editor/src/utils/userNames.js

```diff
@@ -0,0 +1,30 @@
+// Fallback value for user without name set
+export const unknownUserNameValue = "Unknown User";
```

Comment:
Not sure this will ever really happen since we validate first name and last name are present on user creation but it could be good to be ready for abnormal data if it were to ever happen.